pgjdbc/src/main/java/org/postgresql/jdbc/PgConnection.java

```diff
@@ -328,6 +292,39 @@ public TimeZone get() {
     replicationConnection = PGProperty.REPLICATION.get(info) != null;
   }
 
+  private Set<Integer> getBinaryOids(Properties info) throws PSQLException {
```

Comment:
if getOidSet is changed to static, consider changing this method to static.

Reply:
made it